cclib/parser/turbomoleparser.py

```diff
@@ -819,12 +823,160 @@ def extract(self, inputfile, line):
+                # No mult.
+                mult = "???"
+            else:
+                mult = symm_parts[1].capitalize()
+            
+            symmetry = "{}-{}".format(mult, symm_parts[-2].capitalize())
+            self.append_attribute("etsyms", symmetry)
+            
+            # Energy should be in cm-1...
+            energy = utils.convertor(utils.float(line.split()[-1]), 'hartree', 'wavenumber')
+            self.append_attribute("etenergies", energy)
+            
+            # Oscillator strength.
+            while "length representation:" not in line:
+                line = next(inputfile)
+            oscillator_strength = utils.float(line.split()[-1])
+            self.append_attribute("etoscs", oscillator_strength)
+            
+            line = next(inputfile)
+            
+            # Rotatory strength.
+            while "length representation:" not in line:
+                line = next(inputfile)
+            rotatory_strength = utils.float(line.split()[-1])
+            self.append_attribute("etrotats", rotatory_strength)
+            
+            while "Dominant contributions:" not in line:
+                line = next(inputfile)
+            line = next(inputfile)
+            line = next(inputfile)
+            line = next(inputfile)
+            
+            # We can't get transitions if we don't have any orbitals.
+            if hasattr(self, 'moirreps'):
+                transitions = []
+                
+                while len(line.split()) > 0:
+                    parts = line.split()
+                    
+                    # Get alpha/beta, deleting "alpha" or "beta" from the line so
+                    # our indexes are aligned for both RHF and UHF.
+                    if parts[2] == "alpha":
+                        start_AB = 0
+                        parts.pop(2)
+                    if parts[2] == "beta":
+                        start_AB = 1
+                        parts.pop(2)
+                    else:
+                        start_AB = 0
+                
+                    # Determine our start orbital.
+                    start_MO_irrep = (int(parts[0]), self.normalisesym(parts[1]))
+                    start_MO = self.moirreps[start_AB].index(start_MO_irrep)
+                    
+                    if parts[5] == "beta":
+                        end_AB = 1
+                    else:
+                        end_AB = 0
+                        
+                    # And end orbital.
+                    end_MO_irrep = (int(parts[3]), self.normalisesym(parts[4]))
+                    end_MO = self.moirreps[end_AB].index(end_MO_irrep)
+                    
+                    # Finally, get our coefficient.
+                    # Sadly, Turbomole only prints |coeff.|^2*100, so we can't determine whether
+                    # the coefficient is negative or positive...
+                    coeff = (utils.float(parts[-1]) /100) ** 0.5

```

Comment:
I'm not sure this is a good idea, because then the norm over all coefficients will be wrong. It might be better to use `np.nan` instead. Is it still useful as-is?  A logger warning should be printed if this is kept.

Reply:
That's a good point which I hadn't considered :/

Yes it is still useful like this, because the square of the coefficient gives the probability which is (for me personally) more useful than the coefficient anyway. Obviously the probability is independent of the sign of the coefficient, and as far as I know there's no specific cclib attribute for transition probability, so hence the implementation here.

It seems strange to me that Turbomole wouldn't provide the coefficients somewhere, but I can't see it. If there is some other way in which the sign of the coefficient could be determined I guess that would be best, but I don't know how that could be done...

Reply:
I can't find anything in the manual about setting the print level for the TDDFT program.

I am ok with this as long as you add a `logger.warning` call just before setting `etsecs` that the signs of the coefficients are not available, but the probabilities are correct.

Reply:
I've added the warning, might want to tweak the wording?